docs/DafnyRef/Specifications.md

```diff
@@ -479,19 +485,32 @@ able to give invariants to preserve it in this case, it gets even more
 complex when manipulating data structures. In this case, framing is
 essential to making the verification process feasible.
 
-It is not just the body of a function that is subject to `reads`
+By default, methods are not required to list the things they read.
```

Comment:
Made this memory locations - "object or fields" felt a bit weird to me since you can only ever read object fields rather than objects directly, and the latter could imply things like reading the address of an object.